<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="vLayoutParts" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
            <item>
             <widget class="QPushButton" name="btnBack">
              <property name="text">
               <string>back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblModelName">
              <property name="text">
               <string>Models</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnInitialNames">
              <property name="text">
               <string>Initial names</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTreeView" name="treeModelsParts"/>
            </item>
            <item>
             <widget class="QTreeView" name="treeInitParts">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnLoadModel">
            <property name="text">
             <string>Import model</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveChanges">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save changes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="ResizeControlWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <widget class="MaintenanceGL" name="maintenance3D" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="vLayoutDescription">
          <item>
           <widget class="QLabel" name="lblDescription">
            <property name="text">
             <string>Description</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboSectionName">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSectionName"/>
          </item>
          <item>
           <widget class="QTextEdit" name="editSectionDescription"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hLayoutControl" stretch="0,0,1,0">
            <item>
             <widget class="QPushButton" name="btnLoadSection">
              <property name="text">
               <string>Load section</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSaveSection">
              <property name="text">
               <string>Save section</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnDeleteSection">
              <property name="text">
               <string>Delete section</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MaintenanceGL</class>
   <extends>QWidget</extends>
   <header>MaintenanceGL.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResizeControlWidget</class>
   <extends>QWidget</extends>
   <header>ResizeControlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
